requirements/requirements.txt

```diff
@@ -33,6 +33,7 @@ phonenumberslite==6.0.0a
 poster==0.8.1
 psycopg2==2.5.1
 psycogreen==1.0
+pycrypto==2.6.1
```

Comment:
Looks like we already have pycrypto installed in our codebase (must be a dependency for another package), but just pinning it here